specification/cognitiveservices/data-plane/TranslatorText/Batch/preview/v1.0/TranslatorBatch.json

```diff
@@ -0,0 +1,1272 @@
+{
```

Comment:
Any specific reason why this will be a separated swagger and SDK?

Reply:
you mean from the regular translation API?  it's a new product completely.